includes/wcal-functions.php

```diff
@@ -85,3 +87,18 @@ function wcal_get_product_details( $cart_data ) {
 
 	return $product_details;
 }
+/**
+ * Returns if HPOS is enabled
+ *
+ * @return bool
+ * @since  5.14.2
+ */
+function is_hpos_enabled() {
```

Comment:
Please add the plugin prefix to the function. So the new name would be `wcal_is_hpos_enabled()` 

Please relevant changes as needed across the plugin for the same.

This will ensure there's no conflict with any other plugin using a commonly named function such as is_hpos_enabled()